Resources/public/js/views/fields/ez-selection-editview.js

```diff
@@ -125,6 +117,49 @@ YUI.add('ez-selection-editview', function (Y) {
             );
         },
 
+        _unselectSelectionFilter: function () {
```

Comment:
this method is badly named and not documented. It should have a name like `_addSelectionFilterUnselectEventHandler` but is it necessary to add such method ? If you feel it is, you should rename it, do the same for the `select` event and properly document those methods.